libc/test/include/fpclassify_test.c

```diff
@@ -0,0 +1,25 @@
+//===-- Unittests for fpclassify macro ------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDSList-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include "include/llvm-libc-macros/math-function-macros.h"
+
+#include <assert.h>
+
+// check if macro is defined
+#ifndef fpclassify
+#error "fpclassify macro is not defined"
+#else
+int main(void) {
+  assert(fpclassify(1.819f) == 4);
```

Comment:
Use the corresponding `FP_*` constants instead.

Reply:
done.